tests/test_automation/nightly_test_scripts/nightly_olcf_spock.sh

```diff
@@ -3,7 +3,7 @@
 #SBATCH -J nightly_spock
 #SBATCH -o nightly_spock.%j
 #SBATCH -e nightly_spock.%j
-#SBATCH -t 00:15:00
+#SBATCH -t 00:25:00
```

Comment:
@williamfgc I added 5 minutes just be sure ctest have enough minutes to stop at now+20mins